src/main/java/com/google/cloud/resourcemanager/ResourceManager.java

```diff
@@ -337,4 +337,15 @@ public static ProjectListOption fields(ProjectField... fields) {
    *     Platform Services</a>
    */
   List<Boolean> testPermissions(String projectId, List<String> permissions);
+
+  /**
+   * Sets the access control policy on an Organization resource. Replaces any existing policy. The
+   * resource field should be the organization's resource name, e.g. "organizations/123".
+   *
+   * @throws ResourceManagerException upon failure
+   * @see <a href=
+   *     "https://cloud.google.com/resource-manager/reference/rest/v1/organizations/setIamPolicy">
+   *     Resource Manager setIamPolicy</a>
+   */
+  Policy replaceOrgPolicy(String resource, Policy newPolicy);
 }
```

Comment:
"The resource field should be the organization's resource name" is not applied well for the method parameter.
It's better add parameters descriptions:

@param resource the organization's resource name, e.g. "organizations/123" 
@param newPolicy new IAM policy for resource

is it allowed to pass `null` as policy? If yes, it's worth to mention this.

You need to specify what does the method return.

Reply:
done